src/main/java/com/crowdin/cli/Cli.java

```diff
@@ -14,7 +14,7 @@ public static void main(String[] args) {
             Actions actions = new CliActions();
             PropertiesBuilders propertiesBuilders = new PropertiesBuilders();
             int exitCode = picocliRunner.execute(actions, propertiesBuilders, args);
-            if (exitCode != -1 && !picocliRunner.hasMatchedArg("plain")) {
+            if (exitCode != -1 && picocliRunner.noneMatchArgs("help", "verbose", "plain") && args.length != 0) {
```

Comment:
We need to ignore version checking for  '-V' or '--version' parameters but not for `verbose`

Reply:
done